spec.bs

```diff
@@ -504,9 +504,15 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
       1. [=list/For each=] |ig| of |groups|:
         1. TODO: create bidder script runner. If failed, [=iteration/continue=].
         1. TODO: Let |interestGroup| be ... from |ig| ... minus priority and prioritySignalsOverrides and any browser-defined pieces
-        1. TODO: Set |browserSignals|'s "joinCount" to |ig|'s ...need to add and maintain joinCount in interest groups
-        1. TODO: Set |browserSignals|'s "bidCount" to |ig|'s ...need to add and maintain bidCount in interest groups
-        1. TODO: Set |browserSignals|'s "prevWins" to |ig|'s ...need to add and maintain prevWins in interest groups
+        1. TODO: Set |browserSignals|'s "joinCount" to the sum of |ig|'s
+           [=interest group/join counts=] for all days within the last 30 days.
+        1. TODO: Set |browserSignals|'s "bidCount" to the sum of |ig|'s
+           [=interest group/bid counts=] for all days within the last 30 days.
+        1. TODO: Set |browserSignals|'s "prevWins" to a [=list=] containing

```

Comment:
These sound pretty good to me; like they're accurate enough to be spec?  I wonder if we can remove the TODO?  Maybe add Mike Taylor as a reviewer to answer that.

Reply:
Oops, forgot to remove the TODO...